src/test/unit/services/sample/hmc_nuts_diag_inv_metric_test.cpp

```diff
@@ -98,10 +99,9 @@ TEST_F(ServicesSampleHmcNutsDiagEMassMatrix, use_metric_no_adapt) {
   stan::test::unit::instrumented_interrupt interrupt;
   EXPECT_EQ(interrupt.call_count(), 0);
 
-  std::string txt
-      = "inv_metric <- structure(c(0.787405, 0.884987, 1.19869),.Dim=c(3))";
+  std::string txt = R"json({"inv_metric" : [0.787405, 0.884987, 1.19869]})json";
   std::stringstream in(txt);
-  stan::io::dump dump(in);
+  stan::json::json_data dump(in);
```

Comment:
now that were're not using R dump format, is there a better name for user-specified inits and metrics?
why in this file do we have both `dmp` and `dump` objects?